streams-tests/shared/src/test/scala/zio/stream/ZSinkSpec.scala

```diff
@@ -25,6 +25,17 @@ object ZSinkSpec extends ZIOBaseSpec {
             .run(ZSink.collectAllToMap((_: Int) % 3)(_ + _))
         )(equalTo(Map[Int, Int](0 -> 18, 1 -> 12, 2 -> 15)))
       ),
+//      testM("collectAllToMap referential transparancy") {
```

Comment:
Can we decide on the fate of this test? Either it stays uncommented or it gets deleted :-)

Reply:
I'm just not well versed in the zio test libary yet to easily implement it. I need more experience first.